tests/benchdnn/eltwise/eltwise.cpp

```diff
@@ -354,7 +368,8 @@ void setup_cmp(compare::compare_t &cmp, const prb_t *prb, data_kind_t kind,
                 const auto &source
                         = ((prb->dir & FLAG_BWD) && prb->use_dst()) ? dst : src;
                 const float s = source.get_elem(args.idx);
-                if (check_abs_err(prb, s, args.trh))
+                if (check_abs_err(prb, s, args.trh)
+                        || miopen_check_correctness(prb, args))
```

Comment:
Thanks for addressing the feedback! 

Please also replace `return false` in the last line of this lambda with the new `miopen_check_correctness` call, otherwise, the check is not doing what it needs to do:
Current way the function is expected to return `true`/`false` and will check the criterion of `return args.diff <= args.trh;` if `true`, while the desire is to make the function rely on `return args.diff <= epsilon_dt(args.dt);` condition instead which is not happening as of now.

With the change suggested, it would follow the desired approach.

Reply:
I made the suggested change. I hope this is what you mentioned. Thanks @dzarukin.

Reply:
Thank you, resolving!